[2025-09-19T15:15:03.535+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_main.check_new_files scheduled__2025-09-19T15:00:00+00:00 [queued]>
[2025-09-19T15:15:03.548+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_main.check_new_files scheduled__2025-09-19T15:00:00+00:00 [queued]>
[2025-09-19T15:15:03.549+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-09-19T15:15:03.578+0000] {taskinstance.py:2191} INFO - Executing <Task(BranchPythonOperator): check_new_files> on 2025-09-19 15:00:00+00:00
[2025-09-19T15:15:03.585+0000] {standard_task_runner.py:60} INFO - Started process 1765 to run task
[2025-09-19T15:15:03.591+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_main', 'check_new_files', 'scheduled__2025-09-19T15:00:00+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmp400nwdbz']
[2025-09-19T15:15:03.595+0000] {standard_task_runner.py:88} INFO - Job 111: Subtask check_new_files
[2025-09-19T15:15:03.671+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_main.check_new_files scheduled__2025-09-19T15:00:00+00:00 [running]> on host ed735ff6f76b
[2025-09-19T15:15:03.809+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='crypto_etl_main' AIRFLOW_CTX_TASK_ID='check_new_files' AIRFLOW_CTX_EXECUTION_DATE='2025-09-19T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-19T15:00:00+00:00'
[2025-09-19T15:15:03.969+0000] {log.py:232} WARNING - [2025-09-19 15:15:03,968] INFO - crypto_etl - Listing files from bucket crypto-data with prefix 'raw-data/'
[2025-09-19T15:15:03.968+0000] {extract.py:26} INFO - Listing files from bucket crypto-data with prefix 'raw-data/'
[2025-09-19T15:15:03.990+0000] {log.py:232} WARNING - [2025-09-19 15:15:03,989] INFO - crypto_etl - Found 4 CSV files
[2025-09-19T15:15:03.989+0000] {extract.py:30} INFO - Found 4 CSV files
[2025-09-19T15:15:04.002+0000] {log.py:232} WARNING - [2025-09-19 15:15:04,001] INFO - crypto_etl - Found 1 unprocessed files out of 4 total
[2025-09-19T15:15:04.001+0000] {file_processor.py:25} INFO - Found 1 unprocessed files out of 4 total
[2025-09-19T15:15:04.008+0000] {logging_mixin.py:188} INFO - Found 1 new files to process
[2025-09-19T15:15:04.041+0000] {python.py:201} INFO - Done. Returned value was: validate_files
[2025-09-19T15:15:04.042+0000] {branch.py:35} INFO - Branch into validate_files
[2025-09-19T15:15:04.043+0000] {skipmixin.py:178} INFO - Following branch validate_files
[2025-09-19T15:15:04.069+0000] {skipmixin.py:244} INFO - Skipping tasks [('no_new_files', -1)]
[2025-09-19T15:15:04.134+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_main, task_id=check_new_files, execution_date=20250919T150000, start_date=20250919T151503, end_date=20250919T151504
[2025-09-19T15:15:04.206+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-19T15:15:04.270+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
